ts/screens/authentication/InterruptedReadingCardScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import { Container, H2, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type Props = OwnProps;
+
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: variables.contentPadding
+  },
+  text: {
+    fontSize: variables.fontSizeBase
+  }
+});
+
+class InterruptedReadingCardScreen extends React.Component<Props> {
+  /**
+   * Footer
+   */
+  private renderFooterButtons() {
```

Comment:
```suggestion
  get cancelButtonProps() {
    return {
      cancel: true,
      onPress: this.props.navigation.goBack,
      title: I18n.t("global.buttons.cancel"),
      block: true
    };
  }

  get retryButtonProps() {
    return {
      onPress: this.props.onRetry,
      title: I18n.t("global.buttons.retry"),
      block: true
    };
  }

  private renderFooterButtons() {
    return (
      <FooterWithButtons
        type={"TwoButtonsInlineThird"}
        leftButton={this.cancelButtonProps}
        rightButton={this.retryButtonProps}
      />
    );
  }
```
this implementation avoid the creation of **cancelButtonProps** and **retryButtonProps** each time renderFooterButtons is executed

Reply:
Good!